css/main.css

```diff
@@ -18,7 +18,7 @@ hr {
   width: 80vw;
 }
 
-header {
+#header {
   display: flex;
   justify-content: space-between;
   align-items: center;
```

Comment:
- [ ]  Please make sure that you fix the overlap on your mobile menu. as the previous code reviewer mentioned stick to the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) as it is the project requirement of your portfolio.